inference-engine/src/transformations/CMakeLists.txt

```diff
@@ -33,6 +33,8 @@ target_link_libraries(${TARGET_NAME} PUBLIC ${NGRAPH_LIBRARIES}
 target_include_directories(${TARGET_NAME} PUBLIC ${PUBLIC_HEADERS_DIR}
                                           PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/src")
 
+target_compile_definitions(${TARGET_NAME} PRIVATE ngraph_EXPORTS)
```

Comment:
Not sure that it will work for Windows.

I suggest:
1. `class TRANSFORMATIONS_API TypeRelaxedBase` -> `NGRAPH_API`
2. `class TRANSFORMATIONS_API ConvertPrecision` -> `NGRAPH_API`

Reply:
TRANSFORMATIONS_API  works on Windows even without `target_compile_definitions(${TARGET_NAME} PRIVATE ngraph_EXPORTS)`

Reply:
So I would remove target_compile_definitions from /transformations/CMakeLists.txt

Reply:
Since convert precisions is becoming ngraph transfromation (part of ngraph.so, but with header located in a different places), its definitions should be marked with `NGRAPH_API`, not with transformation's one

Reply:
Done.